test/functional/tests/stats/test_consistency_between_outputs.py

```diff
@@ -0,0 +1,299 @@
+        exp_obj = cache.add_core(core_part)
+
+    for _ in TestRun.iteration(range(iterations), f"Do next steps {iterations} times"):
+        with TestRun.step("Reset stats and run workload on the exported object."):
+            exp_obj.reset_counters()
+            # Run workload on a random portion of the cache capacity,
+            # not too small, but not more than half the size
+            random_count = random.randint(blocks_in_cache / 32, blocks_in_cache / 2)
+            dd_builder(cache_mode, cache_line_size, random_count, exp_obj).run()
+
+        with TestRun.step("Flush core and get statistics from different outputs."):
+            exp_obj.flush_core()
+            csv_stats = get_stats_from_csv(cache.cache_id, exp_obj.core_id)
+            table_stats = get_stats_from_table(cache.cache_id, exp_obj.core_id)
+
+        with TestRun.step("Compare statistics between outputs."):
+            if csv_stats != table_stats:
+                TestRun.LOGGER.error(f"Inconsistent outputs:\n{csv_stats}\n\n{table_stats}")
+
+
+def get_stats_from_csv(cache_id: int, core_id: int = None):
+    """
+    'casadm -P' csv output has two lines:
+    1st - statistics names with units
+    2nd - statistics values
+    This function returns dictionary with statistics names with units as keys
+    and statistics values as values.
+    """
+    output = print_statistics(cache_id, core_id, output_format=OutputFormat.csv)
+
+    output = output.stdout.splitlines()
+    #
+    keys = output[0].split(",")
+    values = output[1].split(",")
+
+    # put the keys and the values into a dictionary
+    dictionary_output = {}
+    for i in range(len(keys)):
+        dictionary_output[keys[i]] = values[i]
+
+    return dictionary_output
+
+
+def get_stats_from_table(cache_id: int, core_id: int = None):
+    """
+    'casadm -P' table output has a few sections:
+    1st - config section with two columns
+    remaining - table sections with four columns
+    This function returns dictionary with statistics names with units as keys
+    and statistics values as values.
+    """
+    output = print_statistics(cache_id, core_id, output_format=OutputFormat.table)
+    output = output.stdout.splitlines()
+
+    output_parts = []
+    keys = []
+    values = []
+
+    # split 'casadm -P' output to sections and remove blank lines
+    j = 0
+    for i, line in enumerate(output):
+        if line == "" or i == len(output) - 1:
+            output_parts.append(output[j:i])
+            j = i + 1
+
+    # parse each section
+    for i in range(len(output_parts)):
+        # the first part is config section
+        if i == 0:
+            part_of_keys, part_of_values = (
+                parse_core_conf_section(output_parts[i]) if core_id
+                else parse_cache_conf_section(output_parts[i])
+            )
+        # the remaining parts are table sections
+        else:
+            part_of_keys, part_of_values = parse_tables_section(output_parts[i])
```

Comment:
You can move parsing conf section before loop, remove first value from `output_parts` and use `for section in output_parts:` which seems more pythonic.